lib/Manager.php

```diff
@@ -212,7 +212,7 @@ public function getAnnouncement(int $id, bool $parseStrings = true, bool $ignore
 			'author'	=> $row['announcement_user'],
 			'time'		=> (int) $row['announcement_time'],
 			'subject'	=> $parseStrings ? $this->parseSubject($row['announcement_subject']) : $row['announcement_subject'],
-			'message'	=> $parseStrings ? $this->parseMessage($row['announcement_message']) : $row['announcement_message'],
```

Comment:
this basically makes it possible to run any HTML/JS.
I think this is not what you wanted?

Your idea was to not convert line breaks into `<br />` because the JS lib does this already?

Reply:
Hi @nickvergessen ,
Thanks for heads up.
There is two thing I want to keep, which is `\n` and `>`.
`>` used for the quote.